client/src/legacy/TabSet.js

```diff
@@ -17,6 +17,39 @@ $.entwine('ss', function($){
 
       this._super();
     },
+    
+    onadd: function () {
+        this.on(
+            "tabsactivate",
+            function (event, { newPanel }) {
+                this.triggerLazyLoad(newPanel);
+            }.bind(this)
+        );
+        this.on(
+            "tabscreate",
+            function (event, { panel }) {
+                this.triggerLazyLoad(panel);
```

Comment:
```suggestion
                this.lazyLoadGridFields(panel);
                this.triggerLazyLoad(panel);
```
And remove the `tabscreate` event handler in the `.ss-tabset`-exclusive entwine scope